cli/src/base/ApiCommandBase.ts

```diff
@@ -424,12 +425,16 @@ export default abstract class ApiCommandBase extends StateAwareCommandBase {
     method: string,
     params: CodecArg[],
     warnOnly = false
-  ): Promise<boolean> {
+  ): Promise<SubmittableResult | false> {
     this.log(chalk.magentaBright(`\nSending ${module}.${method} extrinsic...`))
     const tx = await this.getOriginalApi().tx[module][method](...params)
     return await this.sendAndFollowTx(account, tx, warnOnly)
   }
 
+  public findEvent(result: SubmittableResult, section: string, method: string): Event | undefined {
```

Comment:
There is actually a `.findRecord(section, method)` method on the `SubmittableResult` type that i've used here for example https://github.com/Joystream/joystream/blob/master/storage-node/packages/runtime-api/index.js#L336